CMakeLists.txt

```diff
@@ -65,6 +65,9 @@ install(FILES ${CMAKE_CURRENT_BINARY_DIR}/libKeyFinder.pc DESTINATION "${CMAKE_I
 #
 # Tests
 #
-add_subdirectory(tests)
+include(CTest)
 enable_testing()
```

Comment:
Should these be inside the `if` block?

Reply:
No. `include(CTest)` adds the `BUILD_TESTING` option implicitly. But the enable_testing should be removed:

> This command is automatically invoked when the CTest module is included, except if the BUILD_TESTING option is turned off.